mopidy_local/commands.py

```diff
@@ -221,6 +221,10 @@ def _scan_metadata(
                     logger.warning(
                         f"Failed scanning {file_uri}: No audio found in file"
                     )
+                elif result.duration is None:
+                    logger.warning(
+                        f"Failed scanning {file_uri}: No duration information found in file"
```

Comment:
This line fails flak8 linting:

```
./mopidy_local/commands.py:226:93: B950 line too long (92 > 80 characters)
                        f"Failed scanning {file_uri}: No duration information found in file"
                                                                                            ^
1     B950 line too long (92 > 80 characters)
```

You can run all the CI tests locally by running the `tox` command.

Reply:
Thanks, that's fair. I'll force-push an updated version of this commit and test with `tox` before.